src/components/InstructionBlock.tsx

```diff
@@ -0,0 +1,52 @@
+import { CaptionText, InstructionText } from 'components/Text'
+import Instruction from 'models/Instruction'
+import TokenScreenParams from 'models/TokenScreenParams'
+import classnames, {
+  alignItems,
+  backgroundColor,
+  borderRadius,
+  display,
+  flexDirection,
+  gap,
+  justifyContent,
+  maxWidth,
+  padding,
+} from 'classnames/tailwind'
+
+const wrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  justifyContent('justify-center'),
+  alignItems('items-center'),
```

Comment:
You can extract those into `const columnCentered`

Reply:
Done